src/DynamicExpresso.Core/Parameter.cs

```diff
@@ -38,6 +38,11 @@ public Parameter(string name, Type type, object value = null)
 			Expression = System.Linq.Expressions.Expression.Parameter(type, name);
 		}
 
+		public static Parameter Is<T>(string name, T value)
+		{
+			return new Parameter(name, typeof(T), value);
+		}
+
```

Comment:
Maybe rename it to `Create`, or `Make`, or `New`? I'd expect `Is` to return a boolean

Reply:
Changed to Create